src/aof.c

```diff
@@ -2471,7 +2471,7 @@ int rewriteAppendOnlyFileBackground(void) {
         char tmpfile[256];
 
         /* Child */
-        serverSetProcTitle("redis-aof-rewrite");
+        serverSetProcTitle("valkey-aof-rewrite");
```

Comment:
@PingXie You're right that proc title is not logging.

For the most reasonable result we could check if Valkey was started using a `redis-server` symlink, similar to how we check if we was started as `valkey-check-aof` and other names, here: https://github.com/valkey-io/valkey/blob/a5a1377dfc421570046f6c096d9ff35bfb8d003f/src/server.c#L7019

Maybe it's too complex and not important. I'm not sure.

Reply:
>check if Valkey was started using a redis-server symlink

+1

Reply:
I have tried to add a flag similar to server.sentinel_mode, I am still not sure whether it is appropriate and best way.